baseapp/baseapp.go

```diff
@@ -43,6 +43,12 @@ type (
 	StoreLoader func(ms storetypes.CommitMultiStore) error
 )
 
+// MigrationModuleManager is the interface that a migration module manager should implement to handle
+// the execution of migration logic during the beginning of a block.
+type MigrationModuleManager interface {
+	RunMigrationBeginBlock(ctx sdk.Context) bool
```

Comment:
I think this method could return error, right?

Reply:
will return false when [error](https://github.com/cosmos/cosmos-sdk/blob/2878a3c3374371e34d521b1964229327702c7859/types/module/module.go#L709)

Reply:
Isn't error should stop the state machine? There are errors we shouldn't proceed.

Reply:
yes my bad, but can we port the fix to [here](https://github.com/cosmos/cosmos-sdk/pull/17372)